src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -924,6 +946,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
       await waitFor(() =>
         expect(screen.history.push.lastCall.args[0]).to.equal(url),
       );
+      await screen.findByText(/at home/);
```

Comment:
These tests were being flaky for me, because they're only waiting for a url, not actual data to show up.